astropy/nddata/nddata_base.py

```diff
@@ -60,33 +57,13 @@ def wcs(self):
     @abstractproperty
     def meta(self):
         """
-        Metadata, if any, must be dict-like.
+        Meta information, if any.
```

Comment:
@MSeifert04 -- I think it is useful to have them in both places (one of the rare cases where redundancy is a virtue). I think the reason we don't do any checking of dict-like-ness is that we couldn't come up with a general, but easy, way to do it.

There is no way to check whether the user is using masks in the same sense as numpy masks, so the best we can do is let users of both `NDDataBase` and `NDData` know what they should do.

If the comments you moved to `NDData` were not in there before, please keep them there but also add back to `NDDataBase`.